src/go/bootstrap/static/bootstrap.go

```diff
@@ -49,12 +49,6 @@ func ServiceToBootstrapConfig(serviceConfig *confpb.Service, id string, opts opt
 		return nil, err
 	}
 
-	if !opts.DisableTracing {
```

Comment:
https://github.com/GoogleCloudPlatform/esp-v2/blob/6e9e093240a79f9de0f97745c6db37e957b4071c/src/go/commonflags/flags.go#L38

Please categorize by comment which flags are used by bootstrap and which are used by configmanager

Reply:
https://github.com/GoogleCloudPlatform/esp-v2/blob/6e9e093240a79f9de0f97745c6db37e957b4071c/src/go/commonflags/flags.go#L27

All of these *should* be used by bootstrap. But with the changes in this PR, the tracing ones are not.

As mentioned above, I will make a followup PR to clean this up. Don't want to do it in this PR, the diff will be too big.